build-executor/src/main/java/org/jboss/pnc/executor/DefaultBuildExecutionConfiguration.java

```diff
@@ -58,6 +52,7 @@ public DefaultBuildExecutionConfiguration(
         this.scmMirrorRevision = scmMirrorRevision;
         this.scmRevision = scmRevision;
         this.buildType = buildType;
+        this.buildConfigDBId = buildConfigDBId;
```

Comment:
I would leave DB out of name, as from the executor perspective it doesn't have to do anything with db.